experiments/forms.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, *args, **kwargs):
                                                Q(is_template=True)).order_by('name').distinct()
         self.fields['profile'] = ExperimentModelChoiceField(
             queryset=self.profiles,
-            required=True,
+            required=False,
```

Comment:
Is this the portion of the code that is reponsible for requiring an SSH key to submit an experiment?